src/draw/lv_draw_image.c

```diff
@@ -219,6 +219,34 @@ void _lv_draw_image_tiled_helper(lv_draw_unit_t * draw_unit, const lv_draw_image
     lv_image_decoder_close(&decoder_dsc);
 }
 
+void _lv_image_buf_get_transformed_area(lv_area_t * res, int32_t w, int32_t h, int32_t angle, uint16_t scale_x,
+                                        uint16_t scale_y,
+                                        const lv_point_t * pivot)
```

Comment:
No need for the 3rd line for pivot, move `scale_x`, `scale_y`, `pivot` to 2nd line:
```
void _lv_image_buf_get_transformed_area(lv_area_t * res, int32_t w, int32_t h, int32_t angle, 
                                        uint16_t scale_x,  uint16_t scale_y, const lv_point_t * pivot)
```